airflow/contrib/operators/aws_glue_job_operator.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Use correct license header plz

Reply:
@bolkedebruin, the license headers appear to be newly added. Will update shortly.